pro_tes/utils/db.py

```diff
@@ -23,17 +23,17 @@ class DbDocumentConnector:
     """
 
     def __init__(
-        self,
-        collection: Collection,
-        worker_id: str,
+            self,
+            collection: Collection,  # type: ignore
+            worker_id: str,
     ) -> None:
         """Construct object instance."""
-        self.collection: Collection = collection
+        self.collection: Collection = collection  # type: ignore
```

Comment:
Why the `type: ignore`?

Reply:
After removing the deprecated insert, this type: ignore is now not needed. I forgot to remove it, fixed it in the next commit